src/docs/user/references/Notification-service.adoc

```diff
@@ -495,4 +495,65 @@ Presets are not individual event types like the ones described above; instead, t
 | `All with issues`
 | Jobs
 | Automatically adds the events JOB_IN_ERROR, JOB_FINISHED_WITH_ISSUES, TASK_IN_ERROR, TASK_FINISHED_WITH_ERRORS to the subscription
-|===
\ No newline at end of file
+|===
+
+[[microsoft_teams]]
+=== Microsoft teams notifications
+
+To receive Microsoft Teams notifications using the Notification Third-Party notification method, some configuration must be applied in the Microsoft and the Notification Service sides. +
+These configurations are described in the admin guide link:../admin/ProActiveAdminGuide.html#microsoft_teams_configuration[here] and is mandatory to be able to receive Teams notifications.
+
+==== Configure User Settings
+
+To be able to send Teams notification, the Third-Party script needs to connect and retrieve a token using a user's microsoft credentials.
+
+The script reads these credentials from the User Settings.
+
+The Login is provided as plain text but the password must be encrypted before being saved in the User Settings.
+
+You can see how to encrypt the password link:../admin/ProActiveAdminGuide.html#_encrypting_configuration_passwords[Here]
+
+Once the password encrypted:
+
+- Connect to ProActive and go to the <<_glossary_notification_portal,*Notification Portal*>>
+- Click on the Configuration tab then open the Settings drop-down
+- Click on the "Add" button, at the right of the Third-Party input row.
+
+image::notification-user-settings.png[align="center"]
+
+- Provide your microsoft login and you encrypted password with the ENC( prefix separated by a comma
+- Click on save
+
+image::notification-settings-save.png[align="center"]
+
+==== Get microsoft Teams channel URLs
+
+Each subscription that needs to send notification to teams have to provide channel URLs.
+At least one must be provided and it defines the channel(s) where the notification is sent.
+
+You can easily retrieve channels URL from Microsoft Teams desktop App or Web browser:
```

Comment:
You can easily retrieve channel URLs from the Microsoft Teams desktop app or web browser: